packages/arb-token-bridge-ui/src/util/withdrawals/helpers.ts

```diff
@@ -46,6 +47,8 @@ export async function mapETHWithdrawalToL2ToL1EventResult(
 
   return {
     ...event,
+    sender: event.caller,
```

Comment:
(For later) Since going forward our Tx history use cases are getting more complex, should we align our bridge UI's transactions' nomenclature to the one present in the SDK?

 Eg. instead of `sender` and `destinationAddress`, should we also handle our logic around `fromAddress` and `toAddress` to avoid confusion? Or will it create more confusion, what do you all think? (there's also subgraph naming that we'll need to consider, quite a big feat but might worth considering?)

Reply:
Agreed, but it's more of a long-term thing

Reply:
Yeah we should do that at one point. Overall SDK and bridge types/props have much different names, would be nice to make them uniform.